MIT.md

```diff
@@ -1,4 +1,4 @@
-## Copyright 2021, [YOUR NAME]
+## Copyright 2021, Birhanu Gudisa
 
```

Comment:
- [ ] @GutemaG it would be nice fi your MIT license was customized for this particular project, in its current form its more generic and not professional.